utils/test_utils.py

```diff
@@ -13,6 +13,16 @@ def get(api_client, url, status_code=200):
     return response.data
 
 
+def patch_update(api_client, url, data=None, status_code=200):
```

Comment:
you created `utils` folder, because these test utils are now used in multiple apps? I guess, it is the question of personal preference, how to structure these things. I would probably put them in `open_city_profile/tests/utils` to signify that they are common for several apps. BUT a `utils` folder also works.

Reply:
You're right that these could go anywhere. I was perhaps prematurely anticipating that in the future we might add some common code for the apps to use and that they could also live in this `utils` folder. We can definitely move them to a directory that makes more sense to you, but if this is not entirely unsensible, I'd maybe leave it like this?

Reply:
yeah, as I said, matter of preference. common code could live in a separate directory under the name `utils`/`common` or it could live in the directory, where the settings file(s) are. I have seen both approaches in different projects, so totally up to you 👍

Reply:
:+1: let's leave it like this